common/app/views/fragments/topNav/servicesLinks.scala.html

```diff
@@ -48,6 +48,10 @@ <h3 class="popup__group-header">from the guardian:</h3>
                         <a class="brand-bar__item--action" data-link-name="uk : topNav : masterclasses"
                            href="http://www.theguardian.com/guardian-masterclasses?INTCMP=NGW_TOPNAV_UK_GU_MASTERCLASSES">masterclasses</a>
                     </li>
+                    <li class="popup__item">
+                        <a class="brand-bar__item--action" data-link-name="uk : topNav : courses"
+                        href="http://courses.theguardian.com?INTCMP=NGW_TOPNAV_UK_GU_COURSES">courses</a>
```

Comment:
that should be ok, as long as it doesn't conflict with other namespaces at that level?